### ISSUE TITLE
Screen reader is not announcing the response provided by the bot.

### ISSUE THREAD

**Issue by msftedad** [telegramdesktop/tdesktop#27875]
### Steps to reproduce

1. Open Telegram App in Desktop.
2. Open URL: Telegram: Contact @SCTejTestbot on edge.
3. SCTestbot home screen will appear.
4. Turn on Narrator.
5. Navigate to edit field enter any query.
6. Bot response will appear.
7. Observe an issue that Screen reader is not announcing the response provided by the bot.

### Expected behaviour

Screen reader should announce the response provided by the bot after giving any query to the bot.

### Actual behaviour

Screen reader is not announcing the response provided by the bot.

https://github.com/telegramdesktop/tdesktop/assets/93735775/170daf83-2bb0-4f92-a4c6-788fa0cc55b0

### Operating system

22H2 (22621.1105)

### Version of Telegram Desktop

Telegram (4.16.4)

### Installation source

Microsoft Store

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/476
